core/domain/user_services.py

```diff
@@ -1138,9 +1144,9 @@ def update_multiple_users_data(modifiable_user_data_list):
             )
         ):
         user_id = modifiable_user_data.user_id
-        if not user_id:
+        if user_id is None:
             raise Exception(
-                'A valid user id must exist for all the users to be updated.')
+                'Missing user ID.')
```

Comment:
nit: you can move this to the previous line.

Reply:
Done.